docs/generating_coreboot_support_logs.md

```diff
@@ -0,0 +1,176 @@
+Generating coreboot support logs and sending them to verify
+===========================================================
+
+Intro
+-----
+* PLATFORM_IP used as example is `192.168.0.100`
+* PLATFORM_SERIAL_DEVICE used as example is `/dev/ttyUSB0`
+
+Requirements
+------------
+* OS with `cbmem` and `dmesg` installed on the target board
+([`cbmem` installation instruction](https://github.com/pcengines/apu2-documentation/blob/master/docs/cbmem_building.md))
+* Coreboot gerrit account: https://review.coreboot.org/
+* Possibility to login to `root` user on the target board via SSH or enabled
+coreboot boot logs redirection to the serial console
+* Git (type on slave device`apt-get install git -y` to install git on it)
```

Comment:
@acihy slave ? Maybe you meant target ?

Reply:
@pietrushnic Corrected to `target device`.